paparazzi/src/main/java/app/cash/paparazzi/internal/resources/RepositoryLoader.kt

```diff
@@ -120,17 +120,6 @@ abstract class RepositoryLoader<T : LoadableResourceRepository>(
   val resourceFilesAndFolders: Collection<PathString>?,
   val namespace: ResourceNamespace
 ) : FileFilter {
-  /** The set of attribute formats that is used when no formats are explicitly specified and the attribute is not a flag or enum.  */
-  private val DEFAULT_ATTR_FORMATS: Set<AttributeFormat> = Sets.immutableEnumSet(
-    AttributeFormat.BOOLEAN,
-    AttributeFormat.COLOR,
-    AttributeFormat.DIMENSION,
-    AttributeFormat.FLOAT,
-    AttributeFormat.FRACTION,
-    AttributeFormat.INTEGER,
-    AttributeFormat.REFERENCE,
-    AttributeFormat.STRING
-  )
```

Comment:
revert and add `@Suppress("ktlint:standard:property-naming")` to this property.

normally, the change would make sense, but this is a copied class from cs.android.com, and imo, it's convenient to make the java/kotlin sources be as 1:1 readable as possible.